lib/utils/isKeyframeSelector.mjs

```diff
@@ -1,4 +1,9 @@
-import { keyframeSelectorKeywords } from '../reference/keywords.mjs';
+import { keyframeSelectorKeywords, namedTimelineRangeKeywords } from '../reference/keywords.mjs';
+
+const HAS_TIMELINE_RANGE = new RegExp(
+	`^(?:${[...namedTimelineRangeKeywords.values()].join('|')})\\s+(?:\\d+|\\d*\\.\\d+)%$`,
```

Comment:
trivial
note

I wonder if browser support unusual `<number>`s.
e.g. `10e3%`, `-3.4e-2%`

Reply:
Out of scope?

I doubt that anyone writes CSS likes this If anyone does, then they can open an issue and let us know :)